### PR TITLE
Fix/275 fails

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary

Fix for two bugs both found when running the TS in OpenShift. One is related to usage of old consul config and to this bug: https://github.com/quarkusio/quarkus/issues/24277, another is due to the property, which was removed from pom.xml

Please select the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by fedinskiy** [comment]
Since this branch fixes two bugs, it contains two commits

File: properties/pom.xml [hunk 1]
```diff
@@ -32,8 +32,9 @@
             <artifactId>quarkus-config-yaml</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-consul-config</artifactId>
+            <groupId>io.quarkiverse.config</groupId>
+            <artifactId>quarkus-config-consul</artifactId>
+            <version>1.0.2</version>
```
Comment (pjgg) [inline properties/pom.xml:37]: I think that this version value should be on the main pom.xml properties

**Review by pjgg — CHANGES_REQUESTED** [review]

File: sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/openshift/OpenShiftPostgresqlIT.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+import org.junit.jupiter.api.Disabled;
+
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.ts.reactive.PostgresqlDatabaseIT;
+
+@OpenShiftScenario
+//TODO: enable disabled methods then the issue will be fixed
+public class OpenShiftPostgresqlIT extends PostgresqlDatabaseIT {
+    @Override
```
Comment (rsvoboda) [inline sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/openshift/OpenShiftPostgresqlIT.java:11]: https://github.com/quarkusio/quarkus/issues/21700 seems to be fixed in 2.6.2.Final by https://github.com/quarkusio/quarkus/pull/22644

Please check if the tests are passing now and create new PR for drop of  `@Disabled("https://github.com/quarkusio/quarkus/issues/21700")`

File: sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/PostgresqlDatabaseIT.java [hunk 3]
```diff
@@ -7,12 +7,12 @@
 import io.quarkus.test.services.QuarkusApplication;
 
 @QuarkusScenario
-public class Postgresql13DatabaseIT extends AbstractReactiveDatabaseIT {
+public class PostgresqlDatabaseIT extends AbstractReactiveDatabaseIT {
```
Comment (rsvoboda) [inline sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/PostgresqlDatabaseIT.java:10]: Why this rename? There is also `OpenShiftPostgresql10IT.java`. I would really keep the naming as is in this PR.

If you have proposal for DB related TestCases renaming, please bring it up on Monday meeting.

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/PostgresqlDatabaseIT.java (hunk 3 above)
Comment (rsvoboda) [reply sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/PostgresqlDatabaseIT.java:10]: OK, looking more into the code and it probably makes sense to do it this way, it keeps consistency with other tests naming in that module.

Created trello task to discuss the generic naming topic for DB related test cases.

File: properties/pom.xml (hunk 1 above)
Comment (rsvoboda) [reply properties/pom.xml:37]: +1 to parametrize this in the root pom.xml

File: sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/openshift/OpenShiftPostgresqlIT.java (hunk 2 above)
Comment (fedinskiy) [reply sql-db/hibernate-reactive/src/test/java/io/quarkus/ts/reactive/openshift/OpenShiftPostgresqlIT.java:11]: Will do
https://trello.com/c/9mpNtShc/1320-enable-disabled-tests-in-hibernate-reactive

**Review by rsvoboda — APPROVED** [review]

File: properties/pom.xml (hunk 1 above)
Comment (fedinskiy) [reply properties/pom.xml]: Since it is the only usage of this artefact, I assume that this change will bring us more harm, than good, but it's done.

Comment (rsvoboda) [reply properties/pom.xml]: best practice is to have versions defined in one place - root pom.xml

**Pull request merged**